docs/element-definitions.md

```diff
@@ -475,17 +525,16 @@ Type with domain of all finite character strings.
 - `element` - `"string"`
 - `attributes`
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
-    - `fixed` ([String](#string-element)) - Domain restricted to value given in `content`.
+    - `fixed` ([String](#string-element)) - The type this Element describes is restricted to the value given in `content`.
   - `validation` - _reserved for future use_
-  - `samples` ([Array[String]](#array-element)) - array of alternative sample values for this Element; type of items in `samples` MUST match the type of this Element
-  - `default` (String) - default value for this Element; type of `default` MUST match the type of this Element
-- `content` - finite character string
+  - `samples` ([Array[[String](#string-element)]](#array-element)) - Alternative sample values for this Element; type of items in `samples` MUST match the type this Element describes
```

Comment:
Bad linking. Use something similar to `([Array](#array-element)[[String](#string-element)])`